packages/angular-cli/commands/serve.ts

```diff
@@ -1,9 +1,14 @@
 const PortFinder = require('portfinder');
 const Command = require('../ember-cli/lib/models/command');
+import { CliConfig } from '../models/config';
 
-PortFinder.basePort = 49152;
+const config = CliConfig.fromProject();
 
-const defaultPort = process.env.PORT || 4200;
+const defaultLiveReloadPort = config.get('defaults.serve.liveReloadPort') || 49152;
+const defaultPort = process.env.PORT || config.get('defaults.serve.port');
+const defaultHost = config.get('defaults.serve.host') || 'localhost';
```

Comment:
`config.get` should be avoided in the code. It removes typing information as it returns `any`.